src/Nervos/CommonFunc.h

```diff
@@ -0,0 +1,75 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "CellDep.h"
+#include "OutPoint.h"
+#include "../BinaryCoding.h"
+#include "../Data.h"
+#include "../HexCoding.h"
+
+namespace TW::Nervos {
+
+enum AddressType {
```

Comment:
Let's not use naming like `CommonFunc`, `AddressType` can be moved to `Address.h`

Reply:
Done